websites/T/The Odin Project/presence.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable unicorn/prefer-dom-node-text-content */
+const presence = new Presence({
+		//The client ID of the Application created at https://discordapp.com/developers/applications
+		clientId: "1013489969379152022",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/gtBvHQH.png",
+			startTimestamp: browsingTimestamp,
+		},
+		lesson = document.querySelector(
+			"body > div.page-container.lesson > header > div > div > h1"
+		) as HTMLElement | null,
+		course = document.querySelector(
+			"body > div.gradient.odin-dark-bg > div > div:nth-child(1) > a > h1"
+		) as HTMLElement | null,
+		courseTitle = document.querySelector(
+			"body > div.page-container.lesson > header > div > div > a > h2"
+		) as HTMLElement | null;
+
+	switch (document.location.pathname) {
```

Comment:
`document.location.pathname` is used in multiple references across the presence. Instead, you should create `pathname` as a variable above. `{ pathname } = document.location`